src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,138 @@
+package pages;
+
+import engine.config.reader.ConfigReader;
+import enums.CategoryMenuButton;
+import enums.FooterHyperLink;
+import enums.NavbarButton;
+import org.openqa.selenium.*;
```

Comment:
it is recommended not to use wildcard imports, you can disable it in IntelliJ settings: https://www.baeldung.com/intellij-disable-wildcard-import

Reply:
Interesting topic. Settings from the article implemented, thanks!
![image](https://github.com/fnfldx/automationteststore/assets/56260196/6472db00-3c5a-4f96-b7be-ff1444e26e62)